### ISSUE TITLE
backup/restore: Error: ENOENT: no such file or directory, open '/tmp/tmp-47-tBTgpynioohA/pack.json'

### ISSUE THREAD

**Issue by j2l** [saltcorn/saltcorn#873]
I have a tested XML friendly container on another machine with empty db.
I backed up the previous machine "local container" through `/admin/backup` 
It's a 15.9kB zip file
When I try to restore it on the new machine, I get the error `Error: ENOENT: no such file or directory, open '/tmp/tmp-47-tBTgpynioohA/pack.json'`

But the file pack.json is visible into the container:
```
ls -al /tmp/tmp-47-tBTgpynioohA/
total 52
drwx------ 5 root root  4096 Sep 14 13:40 .
drwxrwxrwt 1 root root  4096 Sep 14 13:40 ..
drwxr-xr-x 2 root root  4096 Sep 14 13:40 config
drwxr-xr-x 2 root root  4096 Sep 14 13:40 files
-rw-r--r-- 1 root root   367 Sep 14 13:36 files.csv
-rw-r--r-- 1 root root 26614 Sep 14 13:36 pack.json
drwxr-xr-x 2 root root  4096 Sep 14 13:40 tables
```
Any idea @glutamate ?

Is it possible to remove parts (tables, pack.json, ...) without breaking something?
I tried to "create a pack" and got:
```
TypeError: Cannot read property 'table_id' of undefined
    at /usr/local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/pack.js:41:48
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async /usr/local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/server/routes/packs.js:96:27
```

**Comment by glutamate** [comment]
yes restore broke in 0.5.4, fixing

**Comment by j2l** [comment]
I see, thank you!

On the pack side:
By adding one by one tables and stuff in a pack, I also found that one view (the only one with spaces and dots in its name) was breaking the pack and throwing the `Cannot read property 'table_id' of undefined` error above.
So I guess view names require some sanitation.

**Comment by glutamate** [comment]
fixed in 0.5.5-beta.0 which also includes the test run console xml fix

**Comment by j2l** [comment]
Thank you! 
Closed in advance because I trust you ;)
